modular_robot/revolve2/modular_robot/body/_module.py

```diff
@@ -1,9 +1,16 @@
 from __future__ import annotations
 
+from abc import ABC, abstractmethod
+from typing import Any, Callable
+
+from revolve2.simulation.scene import JointHinge, MultiBodySystem, Pose, RigidBody
```

Comment:
I am not sure if you understood what I meant before so I will elaborate. The modular robot package has two dependencies:
```
[tool.poetry.dependencies]
python = "^3.10"
numpy = "^1.21.2"
```

Here you import from the `revolve-simulation` package, which is not a dependency. This works because you have installed locally and the CI does not perform a check for this. However, when installing the modular robot package on the raspberry py for the real robots (modular robot is required for brain control), the simulation package will not be installed. As such this import will be broken. In short, the building of the robot into a simulation `Scene` must be moved out of the simulation package to somewhere else. Probably the `modular_robot_simulation` package.